webhook/certificates/controller.go

```diff
@@ -41,6 +42,7 @@ func NewController(
 ) *controller.Impl {
 
 	client := kubeclient.Get(ctx)
+	ctx = injection.WithNamespaceScope(ctx, system.Namespace())
```

Comment:
This won't do what you think

Reply:
Well, that's how I parse the comment here. https://github.com/knative/pkg/blob/master/injection/context.go#L30
What will it do then?

Reply:
Well, apparently it'll do what I think it will do, but I have to put the line elsewhere :)